lang/fpc-sources/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                fpc-sources
+version             3.0.4
+categories          lang
+platforms           darwin
+license             GPL-2 LGPL-2
+maintainers         web.de:karl-michael.schindler
+description         Sources of the FreePascal compiler
+long_description    Sources of the compiler, the runtime library (rtl), \
+                    packages and utils. Used by lazarus.
+homepage            http://www.freepascal.org
+master_sites        sourceforge:freepascal
+
+dist_subdir         ${name}/${version}/source
+distfiles           fpc-${version}.source.tar.gz
+checksums           rmd160 40e1be365e69ed30b4114303c6891cbc60d4b8f3 \
+                    sha256 69b3b7667b72b6759cf27226df5eb54112ce3515ff5efb79d95ac14bac742845 \
+
+set worksrcpath     ${workpath}/fpc-${version}
+
+use_configure       no
+build               {}
+
+destroot {
+    xinstall -d                       ${destroot}${prefix}/share/fpcsrc
+    file copy ${worksrcpath}/compiler ${destroot}${prefix}/share/fpcsrc
+    file copy ${worksrcpath}/rtl      ${destroot}${prefix}/share/fpcsrc
+    file copy ${worksrcpath}/packages ${destroot}${prefix}/share/fpcsrc
+    file copy ${worksrcpath}/utils    ${destroot}${prefix}/share/fpcsrc
+    system "chmod -R 755              ${destroot}${prefix}/share/fpcsrc/*"
```

Comment:
to the best of my knowledge install does not work recursively down the path of directories whereas file copy does. That is why i used file copy instead of xinstall. I'll add a corresponding note.